### ISSUE TITLE
Unbound value Time_ns.*

### ISSUE THREAD

**Issue by xin-jin** [janestreet/core#52]
Hi,
When I load toplevel, there are a few errors, and they seem be caused by `core_top`. Here is the output snippet from running ocaml:

```
/home/xin/.opam/4.02.1/lib/core/core_top.cma: loaded
Unbound value Time_ns.Ofday.pp.
Unbound value Time_ns.pp.
Unbound value Time_ns.Option.pp.
Unbound value Time_ns.Span.Option.pp.
Unbound value Time_ns.Span.pp.
```

My .ocamlinit:

```
(* Added by OPAM. *)
let () =
  try Topdirs.dir_directory (Sys.getenv "OCAML_TOPLEVEL_PATH")
  with Not_found -> ()
;;

#use "topfind";;
#thread;;
#camlp4o;;
#require "batteries";;
#require "core.top";;
#require "core.syntax";;
open Core.Std
```

Thanks!

EDIT: installed-version: 112.06.01 [4.02.1]

**Comment by lhiguera** [comment]
I have the same issue.   This message appeared just after I did an "opam update".  The ocaml version did not change (4.02.1).  My .ocamlinit is the same as that of xin-jin.  Thanks!

**Comment by lambdahands** [comment]
I'm also experiencing this issue. I'm using Core `112.06.01` with OCaml `4.02.1`.

**Comment by pveber** [comment]
Same here!

**Comment by jsnider3** [comment]
I'm also having this problem with Core 112.06.01 and OCaml 4.02.1.

**Comment by diml** [comment]
These are warnings for toplevel printers. They appear because `Time_ns` is passing a non-prefixed module name to the functor `Identifiable.Make`. We'll look into fixing that.

**Comment by agarwal** [comment]
This appears to be fixed in 112.06.02.
